cli/internal/fs/package_json.go

```diff
@@ -14,11 +14,10 @@ type TurboConfigJSON struct {
 	// Base Git branch
 	Base string `json:"baseBranch,omitempty"`
 	// Global root filesystem dependencies
-	GlobalDependencies []string `json:"globalDependencies,omitempty"`
-	TurboCacheOptions  string   `json:"cacheOptions,omitempty"`
-	Outputs            []string `json:"outputs,omitempty"`
-	// RemoteCacheUrl is the Remote Cache API URL
-	RemoteCacheUrl string `json:"remoteCacheUrl,omitempty"`
```

Comment:
@jaredpalmer , I notice this `RemoteCacheUrl` is not used anywhere. Is it okay to remove?

Reply:
Yes